core/Lexer.ts

```diff
@@ -50,6 +50,10 @@ export default class Lexer {
       const matched = line.match(regex);
 
       if (matched && matched[0]) {
+        if (!line.startsWith(matched[0])) {
+          throw new Error(`No value on the position ${this.pos} specified`);
```

Comment:
Also you have a pretty good logic of logger, it would be better to use logger for throwing this error as well. The same is for other errors and tracing in this file.

Reply:
Will be fixed in next issues